packages/pages/src/dev/dev.ts

```diff
@@ -1,15 +1,34 @@
 import { createServer } from "./server/server.js";
 import { viteDevServerPort } from "./server/middleware/constants.js";
+import { CommandModule } from "yargs";
 import open from "open";
 
-export default async () => {
-  const [, , ...args] = process.argv;
-
-  if (args.some((arg) => ["local"].includes(arg))) {
-    await createServer(false);
-  } else {
-    await createServer(true);
-  }
+interface DevArgs {
+  scope?: string;
+  local?: boolean;
+}
 
+const handler = async ({ scope, local }: DevArgs) => {
+  await createServer(!local, scope);
   await open(`http://localhost:${viteDevServerPort}/`);
 };
+
+export const devCommandModule: CommandModule<unknown, DevArgs> = {
+  command: "dev",
+  describe: "Runs a custom local development server that is backed by Vite",
+  builder: (yargs) => {
+    return yargs
+      .option("local", {
+        describe: "Disable dynamically generate test data",
+        type: "boolean",
+        demandOption: false,
+      })
+      .option("scope", {
+        describe:
+          "The subfolder to scope a build to a subset of templates using specific sites-config folder.",
```

Comment:
This sentence is confusing. Why is the sites-config even relevant for the dev command? It's not.